src/main/java/seedu/address/logic/commands/SelectDeckCommand.java

```diff
@@ -44,7 +44,7 @@ public CommandResult execute(Model model) throws CommandException {
         }
 
         model.selectDeck(deckIndex);
-        return new CommandResult(String.format(MESSAGE_SUCCESS, model.getSelectedDeck().getDeckName()));
+        return new CommandResult(String.format(MESSAGE_SUCCESS, model.getSelectedDeck().get().getDeckName()));
```

Comment:
To follow law of demeter (prevent objects from navigating the internal structures of other objects) we can consider making getSelectedDeckName a method in model rather than accessing the selectedDeck then calling its function.